frontend/public/components/utils/units.js

```diff
@@ -222,6 +224,24 @@ export const humanizeCpuCores = (v) => {
     value,
   };
 };
+export const humanizeCpuCoresLong = (v) => {
+  const humanized = humanizeCpuCores(v);
+  const transformedUnit = humanized.unit === 'm' ? ' millicores' : ' cores';
+  return {
+    ...humanized,
+    string: `${formatValue(humanized.value)}${transformedUnit}`,
+  };
+};
+export const humanizeCpuCoresCompact = (v) => {
+  // keep k8s values but round to 3 decimal digits
+  const value = Math.round(v * 1000) / 1000;
```

Comment:
What's wrong with `formatValue` which uses `Intl.NumberFormat`? You can set `maximumFractionDigits` by passing `opts`.

Reply:
Not relevant anymore in the context of comments bellow.